src/test/fixture/ebl-endorsement-chain.json

```diff
@@ -1 +1 @@
-{"version":"https://schema.openattestation.com/2.0/schema.json","data":{"shipper":{"address":{}},"consignee":{},"notifyParty":{},"blNumber":"aa72090d-8643-4bd9-aa75-1fb9a4ac51d9:string:0x8e87c7cEc2D4464119C937bfef3398ebb1d9452e","$template":{"type":"6d7c9eb7-a98f-4831-8d9d-7d112c630ad3:string:EMBEDDED_RENDERER","name":"56fcff70-5482-4955-96e0-8384f37eaecf:string:BILL_OF_LADING","url":"5c92129b-9218-4afa-bb86-711e420b9ec3:string:https://demo-cnm.openattestation.com"},"issuers":[{"identityProof":{"type":"e835c7fd-8c94-4efc-97f5-b1f1e9197717:string:DNS-TXT","location":"03ad335e-7a45-4e15-a4d4-75f34a379545:string:demo-tradetrust.openattestation.com"},"name":"d502b087-5a16-49af-980a-c679adb0e8d1:string:DEMO STORE","tokenRegistry":"d61829b3-2c90-4603-b2fe-4117b5ad0ef7:string:0x10E936e6BA85dC92505760259881167141365821"}],"name":"64f32908-dcda-44b6-992b-f85b6450fe50:string:Maersk Bill of Lading"},"signature":{"type":"SHA3MerkleProof","targetHash":"b6a773c9f8de646207b63154b54fba599a7151c1274823923ed1a80dabfab067","proof":[],"merkleRoot":"b6a773c9f8de646207b63154b54fba599a7151c1274823923ed1a80dabfab067"}}
\ No newline at end of file
+{"version":"https://schema.openattestation.com/2.0/schema.json","data":{"shipper":{"address":{}},"consignee":{},"notifyParty":{},"$template":{"type":"02e6ff2c-71af-4972-a238-74b49ee3ae57:string:EMBEDDED_RENDERER","name":"ee49eb19-67b6-422f-9ee3-d2a95d0e2500:string:BILL_OF_LADING","url":"2a36c4c4-0028-4968-9518-05689f0fb2da:string:https://generic-templates.tradetrust.io"},"issuers":[{"identityProof":{"type":"628e7932-9354-43d4-8eea-7065e0e7a7f1:string:DNS-TXT","location":"7e1b5a94-9626-4639-bbff-f1c3ffa4bfa9:string:demo-tradetrust.openattestation.com"},"name":"280f2c74-243f-4eb3-b433-8182023dba3e:string:DEMO STORE","tokenRegistry":"12d5809f-abe7-4a48-a8e3-8cb716236ba7:string:0x13249BA1Ec6B957Eb35D34D7b9fE5D91dF225B5B"}],"name":"8aa455a0-8d72-4f7a-be00-e0450d3f0216:string:Maersk Bill of Lading","blNumber":"a61437bf-00aa-480a-b25a-dfe45574c710:string:123333","links":{"self":{"href":"653ce601-2ecd-49c0-baae-75c25290b793:string:https://action.openattestation.com?q=%7B%22type%22%3A%22DOCUMENT%22%2C%22payload%22%3A%7B%22uri%22%3A%22https%3A%2F%2Fapi-ropsten.tradetrust.io%2Fstorage%2Fd7263c6f-ed0a-400f-91e8-74c27b717640%22%2C%22key%22%3A%226b0def67903ca79be688e62ef18c74dd293cc18fa0a2aa5c24a038b6d4056c7b%22%2C%22permittedActions%22%3A%5B%22STORE%22%5D%2C%22redirect%22%3A%22https%3A%2F%2Fdev.tradetrust.io%2F%22%7D%7D"}}},"signature":{"type":"SHA3MerkleProof","targetHash":"efe67a6793b95fe9b8c9e2afeb4439482f0970761508d020000803fb6752d725","proof":[],"merkleRoot":"efe67a6793b95fe9b8c9e2afeb4439482f0970761508d020000803fb6752d725"}}
```

Comment:
able to format this? `shift + opt + f` i think

Reply:
ok